pySDC/projects/PinTSimE/battery_implicit.py

```diff
@@ -0,0 +1,162 @@
+import numpy as np
```

Comment:
Same here. This looks a lot like your battery model, so you could simply merge those two, can't you? This way to you can "guarantee" that both scripts run the same setup, just using a different sweeper (and an adapted problem class).

Reply:
You're right. I can. This would save lot of (unnessecary) files..